### ISSUE TITLE
Inconsistent results from multiple queries

### ISSUE THREAD

**Issue by aluode99** [VictoriaMetrics/VictoriaMetrics#5275]
### Describe the question in detail

 Inconsistent results from multiple queries within a specified time range.
![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/15179646/d3a28b23-e921-48ce-95e4-c4f2c09d7e17)

![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/15179646/0f5c8ca5-641e-4d55-bbd5-4b31ad8dfcea)

![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/15179646/cfe01cb1-c583-4c21-ae30-a6e8a5794772)

Consistent results when querying by servicer：
![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/15179646/b21221e6-0004-4e46-be84-ce0501a10173)
![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/15179646/59974dc5-0ffa-4d9b-aa3c-843036a1c1a7)
I tried setting the query parameter nocache=true. When querying multiple servicers, the results are still inconsistent. The results are consistent when querying a single servicer, which can rule out the impact of cache.
     vm is v1.79.0-cluster, vminsert replicationFactor is 2 and vmselect replicationFactor is default(1). The issue of inconsistent query results has been bothering me for a long time. Does the official have any good suggestions?

### Troubleshooting docs

- [ ] General - https://docs.victoriametrics.com/Troubleshooting.html
- [ ] vmagent - https://docs.victoriametrics.com/vmagent.html#troubleshooting
- [ ] vmalert - https://docs.victoriametrics.com/vmalert.html#troubleshooting

**Comment by hagen1778** [comment]
Hi @aluode99!
` v1.79.0-cluster` is out of support. Even if it contains a bug, there is no way the fix can be delivered to that version line. I'd recommend trying to update to supported versions and see if the issue still remains.

**Comment by aluode99** [comment]
> Hi @aluode99! ` v1.79.0-cluster` is out of support. Even if it contains a bug, there is no way the fix can be delivered to that version line. I'd recommend trying to update to supported versions and see if the issue still remains.

Thank you for your reply. I tried adding the 'deny_partial_response' parameter, and I can see that some queries are returning errors. When the results are returned normally, the query results are consistent.

**Comment by hagen1778** [comment]
Right! p.4 from https://docs.victoriametrics.com/Troubleshooting.html#unexpected-query-results :-)
